### ISSUE TITLE
Multiple pixel account

### ISSUE THREAD

**Issue by kossa** [weblagence/laravel-facebook-pixel#3]
Hello,

There is a way to send event to multiple pixel account ?

**Comment by kossa** [comment]
Any answer ?

**Comment by JeremyBellaiche** [comment]
Hi @kossa ,

I just released the v1.1 version which supports now multiple facebook pixels simply by changing your configuration file


```
<?php

return [
    
    /*
     * The Facebook Pixel id, should be a code that looks something like "XXXXXXXXXXXXXXXX".
     */
    'facebook_pixel_id' => env('FACEBOOK_PIXEL_ID', ''),
    
    /*
     * Use this variable instead of `facebook_pixel_id` if you need to use multiple facebook pixels
     */
    'facebook_pixel_ids' => [],
    
    /*
     * Enable or disable script rendering. Useful for local development.
     */
    'enabled'           => true,

    'csp_callback'      => '',
];
```

Think about changing your composer.json 

```
        "weblagence/laravel-facebook-pixel": "^1.1"
```

**Comment by kossa** [comment]
Excellent thank you @JeremyBellaiche
